strawberry/channels/context.py

```diff
@@ -12,6 +12,7 @@ class StrawberryChannelsContext:
     """
 
     request: "ChannelsConsumer"
+    connection_params: Optional[Any] = None
```

Comment:
We actually have a guarantee that it'll be a `Optional[Dict[str, Any]]`, since that's [specified as part of the underlying protocol](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#connectioninit) :)

Reply:
Great news with regards to the established spec, I intended to keep it as general as possible to allow `Any` everywhere but that clearly misaligns with what I wrote elsewhere. I will converge on having `Optional[Dict[str, Any]]` as that is how you describe it here as part of the spec and in other comments.